File: robocode-tankroyale-bot-api-java/src/main/java/dev/robocode/tankroyale/botapi/events/WonRoundEvent.java
package dev.robocode.tankroyale.botapi.events;

/** Event occurring when a bot has won the round. */
public final class WonRoundEvent extends Event {

  /**
   * Initializes a new instance of the WonRoundEvent class.
   *
   * @param turnNumber is the turn number when the bot won the round.
   */
  public WonRoundEvent(int turnNumber) {
    super(turnNumber);
  }
}

File: robocode-tankroyale-bot-api-dotnet/src/events/WonRoundEvent.cs
using Newtonsoft.Json;

namespace Robocode.TankRoyale.BotApi.Events
{
  /// <summary>
  /// Event occurring when a bot has won the round.
  /// </summary>
  public sealed class WonRoundEvent : BotEvent
  {
    /// <summary>
    /// Initializes a new instance of the WonRoundEvent class.
    /// </summary>
    /// <param name="turnNumber">Turn number.</param>
    [JsonConstructor]
    public WonRoundEvent(int turnNumber) : base(turnNumber) { }
  }
}